lib/Controller/RoomController.php

```diff
@@ -210,6 +211,37 @@ public function getRooms(int $noStatusUpdate = 0): DataResponse {
 		return new DataResponse($return, Http::STATUS_OK, $this->getTalkHashHeader());
 	}
 
+	/**
+	 * Search listed rooms
+	 *
+	 * @NoAdminRequired
+	 *
+	 * @param string $searchTerm search term
+	 * @return DataResponse
+	 */
+	public function getListedRooms(string $searchTerm = ''): DataResponse {
+		$event = new UserEvent($this->userId);
+		$this->dispatcher->dispatch(self::EVENT_BEFORE_LISTED_ROOMS_GET, $event);
+
+		$rooms = $this->manager->getListedRoomsForUser($this->userId, $searchTerm);
+
+		$return = [];
+		foreach ($rooms as $room) {
+			try {
+				$roomData = $this->formatRoom($room, null);
+				// since formatRoom will break early due to having no participant,
+				// we populate the remaining base attributes here
+				$return[] = $this->populateBaseRoomData($roomData, $room, $this->userId);
+				// TODO: should we populate more ?
```

Comment:
yeah lets stick to the normal output please. Same array as with normal conversations. Otherwise we have follow up regressions with search results not having the same attributes as conversations from the conversation list

Reply:
we do have the normal output but with zeroed values, as some cannot be populated before we join

I'll check if there's more that we can populate though